themes/default/content/learn/embedding-pulumi/building-api/index.md

```diff
@@ -0,0 +1,328 @@
+---
+title: "Building a Pulumi API"
```

Comment:
I think this title is a bit confusing because there are so many APIs associated with Pulumi that calling this a Pulumi API might not be getting your core point across

Reply:
fixed